config/locales/models/en.yml

```diff
@@ -109,3 +109,5 @@ en:
         user_defined_tags_count: "Fandom, relationship, and character tags"
       work:
         user_defined_tags_count: "Fandom, relationship, character, and additional tags"
+  subscriptions:
+    deleted: "DELETED"
```

Comment:
The placeholders in history use "Deleted work," so let's go with "Deleted item" here.